### PR TITLE
Added Testimonial + 404 Image Added + README Creation

### PR DESCRIPTION
**Pull request by TurboProgramming**
- Created another Testimonial
- Image added to the 404 page.
- Created README.md

### CODE REVIEW

File: 404.html [hunk 1]
```diff
@@ -18,6 +18,11 @@
 </head>
 
 <body>
+    <div class="cover">
+        <center><img src="coding.png" width="275dp"></center>
```
Comment (conorshipp) [inline 404.html]: This image doesn't exist. Did you forget to upload it?

File: 404.html [hunk 2]
```diff
@@ -18,6 +18,11 @@
...
+
```
Comment (conorshipp) [inline 404.html]: Can you remove this blank line?

**Comment by pavi2410** [comment]
Can you wait until I make testimonial section a carousel so that it can accommodate more cards?

**Comment by TurboProgramming** [comment]
@pavi2410 OK, waiting for you. Once, you're done please inform me here so I can add mine. Also suggest you to get other users' opinions using the community :)

File: 404.html (hunk 1 above)
Comment (TurboProgramming) [reply 404.html]: is it OK now?

File: 404.html (hunk 2 above)
Comment (TurboProgramming) [reply 404.html]: done.

**Comment by TurboProgramming** [comment]
@pavi2410 any updates?

**Comment by pavi2410** [comment]
Can you give the github pages url for this?

File: 404.html [hunk 3]
```diff
@@ -16,12 +16,14 @@
...
 </head>
-
 <body>
+    <div class="cover">
+        <center><img src="/assets/coding.png" width="275dp"></center>
```
Comment (pavi2410) [inline 404.html]: Use relative link to assets

**Comment by pavi2410** [comment]
@TurboProgramming I made that. You can take a look at here https://pavi2410.tk/makeroid.github.io/#testimonials

**Comment by TurboProgramming** [comment]
@pavi2410 nice, liked the design. Good Job. 👍 

Just a problem, Carlos Pedroza's testimonial is long that it looks very bad compared to others. The card gots bigger and that kills the carousel animation. Can you make it shorter?

File: 404.html (hunk 3 above)
Comment (TurboProgramming) [reply 404.html]: by relative link what do you mean?

Comment (TurboProgramming) [reply 404.html]: I've uploaded the image to the assets folder: https://github.com/TurboProgramming/makeroid.github.io/blob/master/assets/coding.png

**Comment by pavi2410** [comment]
Yeah, I too thought that. I'll truncate it.

**Comment by TurboProgramming** [comment]
@pavi2410 Let me know when you're done.

File: 404.html (hunk 3 above)
Comment (pavi2410) [reply 404.html]: I mean that relative link won't work across other websites like my fork. https://pavi2410.tk/makeroid.github.io

I've replaced most of the links. You just have to remove the "/" from the beginning

Comment (barreeeiroo) [reply 404.html]: That image doesn't look good there, IMHO

File: README.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# Makeroid Official Website
```
Comment (barreeeiroo) [inline README.md]: If you could add a little bit more information such us screenshots or what we do it would be 👌

File: _data/testimonials.yml [hunk 5]
```diff
@@ -9,3 +9,7 @@
 - name: Carlos Pedroza
   username: carlospedroza27
   text: Makeroid is merging the ability to create awesome android applications without writing a single line of code and the ability to make a professional app in a stunning way.
+
```
Comment (barreeeiroo) [inline _data/testimonials.yml]: I think that due to testimonials design there's a limit of 3
But I'll see if we can display randomly

File: README.md (hunk 4 above)
Comment (TurboProgramming) [reply README.md]: OK, I'm going to try my best.

File: _data/testimonials.yml (hunk 5 above)
Comment (TurboProgramming) [reply _data/testimonials.yml]: @pavi2410 made it a carousel, have you seen it?

https://pavi2410.tk/makeroid.github.io/#testimonials

**Comment by barreeeiroo** [comment]
Already merged

**Pull request closed without merging**